hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -243,8 +243,10 @@ public void run() {
       log.error(msg, e);
       throw new SchedulerException(msg, e);
     }
-
-    registerFramework(this, frameworkInfo.build(), config.getMesosMasterUri());
```

Comment:
Please use Capitalization in the log message. Also, is it consistent with the rest of our logging style?